From eb213d7d2ffe474de9c9eb6f23cbdff58fd739f1 Mon Sep 17 00:00:00 2001
From: JeremyNixon <jnixon2@gmail.com>
Date: Mon, 15 Feb 2016 09:25:13 +0000
Subject: [PATCH 166/201] [SPARK-13312][MLLIB] Update java
 train-validation-split example in ml-guide

Response to JIRA https://issues.apache.org/jira/browse/SPARK-13312.

This contribution is my original work and I license the work to this project.

Author: JeremyNixon <jnixon2@gmail.com>

Closes #11199 from JeremyNixon/update_train_val_split_example.

(cherry picked from commit adb548365012552e991d51740bfd3c25abf0adec)
Signed-off-by: Sean Owen <sowen@cloudera.com>
(cherry picked from commit 71f53edc0e39bc907755153b9603be8c6fcc1d93)
---
 docs/ml-guide.md |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/ml-guide.md b/docs/ml-guide.md
index 6d96596..f5d764c 100644
--- a/docs/ml-guide.md
+++ b/docs/ml-guide.md
@@ -870,7 +870,7 @@ import org.apache.spark.ml.regression.LinearRegression
 import org.apache.spark.ml.tuning.{ParamGridBuilder, TrainValidationSplit}
 
 // Prepare training and test data.
-val data = sqlContext.read.format("libsvm").load("data/mllib/sample_libsvm_data.txt")
+val data = sqlContext.read.format("libsvm").load("data/mllib/sample_linear_regression_data.txt")
 val Array(training, test) = data.randomSplit(Array(0.9, 0.1), seed = 12345)
 
 val lr = new LinearRegression()
@@ -913,7 +913,7 @@ import org.apache.spark.ml.regression.LinearRegression;
 import org.apache.spark.ml.tuning.*;
 import org.apache.spark.sql.DataFrame;
 
-DataFrame data = jsql.read().format("libsvm").load("data/mllib/sample_libsvm_data.txt");
+DataFrame data = jsql.read().format("libsvm").load("data/mllib/sample_linear_regression_data.txt");
 
 // Prepare training and test data.
 DataFrame[] splits = data.randomSplit(new double[] {0.9, 0.1}, 12345);
-- 
1.7.9.5

